app/ducks/namesReducer.js

```diff
@@ -9,6 +9,7 @@ const ALLOWED_COVENANTS = new Set([
   'REGISTER',
   'RENEW',
   'REDEEM',
+  'TRANSFER',
```

Comment:
I forget what this enum is for, but probably needs FINALIZE here also

Reply:
It's used to keep track of which covenants the wallet registers pending transactions for.